ramp-database/ramp_database/tools/submission.py

```diff
@@ -415,18 +425,26 @@ def get_time(session, submission_id):
     computation_time : pd.DataFrame
         A pandas dataframe containing the computation time of each fold.
     """
-    results = defaultdict(list)
-    all_cv_folds = (session.query(SubmissionOnCVFold)
+    cache_filename = os.path.join(
+            CACHE_DIR, 'times_submission_{:09}.pkl.zip'
+            .format(submission_id))
+    if os.path.exists(cache_filename):
+        times = pd.read_pickle(cache_filename)
```

Comment:
Not really sure what is going on here, could you explain? (and also if some of this wasn't addressed in some of the other PRs).